portfolio/src/main/webapp/style.css

```diff
@@ -14,15 +14,27 @@ body {
   background: black;
   margin-left: auto;
   margin-right:auto;
+  /*margin-left: auto;
```

Comment:
delete your commented out code so its cleaner :)

Reply:
Ive made this change :)